src/common/src/ConfirmDialog/ConfirmDialog.test.tsx

```diff
@@ -51,4 +54,35 @@ describe('<ConfirmDialog/>', () => {
 		userEvent.click(confirmButton);
 		expect(onConfirmMock).toBeCalled();
 	});
+
+	it('should call the onClose function when clicking the closeButton', () => {
+		const confirmButton = getByTestId('modal-close-button');
+		userEvent.click(confirmButton);
+		expect(onCloseMock).toBeCalled();
+	});
 });
+
+describe('<ConfirmDialog />', () => {
+	test('Should not show close button',  () => {
+		const props = {
+			title: 'TestDialog',
+			message: 'Are you sure you want to delete this file?',
+			textCancelButton: 'Cancel',
+			textConfirmButton: 'Confirm',
+			onCancel: jest.fn(),
+			onConfirm: jest.fn(),
+			onCloseButton: jest.fn()
+		} as IState;
+
+		render(
+			<div>
```

Comment:
It's ok since this is test code, but in general I'd suggest to always stick to using a fragment rather than a div in such scenarios.

Reply:
Done